setuptools/dist.py

```diff
@@ -96,17 +96,17 @@ def write_pkg_file(self, file):
         file.write('Keywords: %s\n' % keywords)
 
     if version >= StrictVersion('1.2'):
-        for platform in self.get_platforms():
+        for platform in sorted(self.get_platforms()):
             file.write('Platform: %s\n' % platform)
     else:
-        self._write_list(file, 'Platform', self.get_platforms())
+        self._write_list(file, 'Platform', sorted(self.get_platforms()))
```

Comment:
I think all of these `_write_list` calls take a `list` as input, so its order will be reproducible.